### ISSUE TITLE
one of the vote sites is down

### ISSUE THREAD

**Issue by Jerryatric** [dalaranwow/dalaran-wow#5273]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
visit http://www.dalaran-wow.com/account/vote/
the link for Mmorpg top list takes you to an expired domain
**Expected behaviour**: Tell us what should happen instead
expected to vote
**Steps to reproduce the problem**:

1. visit http://www.dalaran-wow.com/account/vote/
2. click on Mmorpg top list
3. it shows you an expired domain and you cannot vote
![mmorpgtoplist](https://user-images.githubusercontent.com/35730907/35276504-9b0a288c-0011-11e8-9ff6-325ee7718dd6.png)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. (Or create a new Ingame-Account in order to be able to talk with a gamemaster with you new level 1 character) Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
